packages/web/pages/about.tsx

```diff
@@ -11,7 +11,7 @@ export default function LandingPage(): JSX.Element {
         description="Omnivore is the free, open source, read-it-later app for serious readers."
       />
 
-      <About lang="en" />
+      <About lang="zh" />
```

Comment:
ok this is the part that changed the about page to Chinese

Reply:
lol, that'll do it. Sorry, must've been when I was trying to figure out how the translations were working (this didn't change it to Chinese locally), and forgot to revert.